cognite/client/data_classes/sequences.py

```diff
@@ -33,6 +34,78 @@
 
     from cognite.client import CogniteClient
 
+ValueType: TypeAlias = Literal["String", "Double", "Long"]
+
+
+class SequenceColumn(CogniteResource):
+    """This represents a column in a sequence. It is used for both read and write.
+
+    Args:
+        external_id (str | None): The external ID provided by the client. Must be unique for the resource type.
+        name (str | None): Name of the column
+        description (str | None): Description of the column
+        value_type (ValueType): The type of the column. Can be String, Double or Long.
+        metadata (dict[str, Any] | None): Custom, application specific metadata. String key -> String value. Maximum length of key is 32 bytes, value 512 bytes, up to 16 key-value pairs.
+        created_time (int | None): Time when this column was created in CDF in milliseconds since Jan 1, 1970.
+        last_updated_time (int | None): The last time this column was updated in CDF, in milliseconds since Jan 1, 1970.
+    """
+
+    def __init__(
+        self,
+        external_id: str | None = None,
+        name: str | None = None,
+        description: str | None = None,
+        value_type: ValueType = "Double",
+        metadata: dict[str, Any] | None = None,
+        created_time: int | None = None,
+        last_updated_time: int | None = None,
+    ) -> None:
+        self.external_id = external_id
+        self.name = name
+        self.description = description
+        self.value_type = value_type
+        self.metadata = metadata
+        self.created_time = created_time
+        self.last_updated_time = last_updated_time
+
+    @classmethod
+    def load(cls, resource: dict | str, cognite_client: CogniteClient | None = None) -> Self:
+        resource = json.loads(resource) if isinstance(resource, str) else resource
+        # Snake case is supported for backwards compatibility
+        resource = convert_all_keys_to_camel_case(resource)
+        return super().load(resource, cognite_client)
+
+    def as_write(self) -> Self:
```

Comment:
I don't love the name `as_write`. Maybe `as_writeable`, although I don't love that either. 

I also think that we should introduce this across the board if we do introduce it, but we can perhaps do it after v7? As far as I can tell it's not breaking.

Reply:
You are right it is not breaking, and it might be good to wait. Will remove it.